src/dcap_provider.cpp

```diff
@@ -745,7 +763,7 @@ static std::string get_issuer_chain_cache_name(std::string url)
 static quote3_error_t get_collateral(
     CollateralTypes collateral_type,
     std::string url,
-    const char header_name[],
+    const char issuer_chain_header[],

```

Comment:
Silly question: Why is this a `const char []` and not a `const char *`?

Reply:
I don't have a preference either way - changed to const char *